### PR TITLE
(maint) update from modulesync

### PR DESCRIPTION
**Pull request by DavidS**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
-script: 'bundle exec rake validate lint spec SPEC_OPTS="--format documentation"'
-
+language: ruby
+bundler_args: --without system_tests
+script: "bundle exec rake validate lint spec SPEC_OPTS='--format documentation'"
 matrix:
   fast_finish: true
-  includes:
+  include:
+  - rvm: 1.8.7
```
Comment (bmjen) [inline .travis.yml]: Is this ruby 1.8.7 node still desired? I've been removing them from other modules.

Comment (DavidS) [inline .travis.yml]: Good question. Regardless of answer, https://github.com/puppetlabs/modulesync_configs/blob/master/config_defaults.yml#L5 is the place to fix it.

**Comment by tphoney** [comment]
@DavidS @bmjen is this still active ?

**Comment by DavidS** [comment]
No.

**Comment by DavidS** [comment]
(and if I am wrong, it'd be re-done anyways with a current config)

**Pull request closed without merging**